.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ jobs:
             - go-mod-v4-{{ checksum "go.sum" }}
       - run:
           name: Compile Binaries
-          command: make build
+          command: make lin/build
```

Comment:
I don't approve that :p

Reply:
yeah - @soc1c I don't see anything wrong with this

Reply:
btw it is functionally the same as the state of the branch from your changes, just `make build` is now the option that doesn't specify the build flags so anyone on mac or linux can just use that to build instead of having to do the OS specific.

Reply:
just trolling, go ahead.

Reply:
:/